src/tuner/nccl_ofi_tuner.c

```diff
@@ -353,15 +393,63 @@ ncclResult_t nccl_ofi_tuner_destroy(void *context)
 	return ncclSuccess;
 }
 
-const ncclTuner_v2_t ncclTunerPlugin_v2 = {
-	.name = "nccl_ofi_tuner", .init = nccl_ofi_tuner_init, .getCollInfo = nccl_ofi_tuner_get_coll_info, .destroy = nccl_ofi_tuner_destroy};
+const ncclTuner_v3_t ncclTunerPlugin_v3 = {.name = "nccl_ofi_tuner",
+					   .init = nccl_ofi_tuner_init,
+					   .getCollInfo = nccl_ofi_tuner_get_coll_info,
+					   .destroy = nccl_ofi_tuner_destroy};
 
-#if !defined(AWS_OFI_NCCL_MIN_TUNER_COMPAT) || (AWS_OFI_NCCL_MIN_TUNER_COMPAT <= 1)
```

Comment:
I was a bit confused by how this is supposed to be used, but anyway I don't see why we need `AWS_OFI_NCCL_MIN_TUNER_COMPAT`, instead of just exporting all symbols. Let me know if you see a need for it.